### ISSUE TITLE
conflicting DNS svchost.exe

### ISSUE THREAD

**Issue by Rexadev** [safing/portmaster#796]
![image](https://user-images.githubusercontent.com/62152714/184474976-d0475125-9853-49d5-8767-2e5e8326fe84.png)
Windows 10 x64 Enterprise 21H1

**Comment by dhaavi** [comment]
This is interesting. What Windows version are you running?

**Comment by Rexadev** [comment]
> This is interesting. What Windows version are you running?

Windows 10 x64 Enterprise 21H1

**Comment by omblivion** [comment]
Having the same error here

**Comment by ahmgam** [comment]
same issue here, using Windows 11 pro 22H2

**Comment by danstis** [comment]
I have the same issue on Win11 22H2.

**Comment by Raphty** [comment]
@dhaavi what info can should people send in this case? it seems to not be a super frequent thing though

**Comment by danstis** [comment]
By digging through my Debug info (in the questionmark logo 3rd from bottom, then scroll to the bottom), I found that I had a conflicting service running on Port 53. 
I then checked the running service using PowerShell:
```powershell
netstat -ano | sls :53
```
I found a running service listening on port 53: `TCP    0.0.0.0:53             0.0.0.0:0              LISTENING       12000`.
From this we can take the last number here which is the process ID of the running app and resolve it to the application using this from PowerShell:
```powershell
Get-Process -Id 12000
```
This then informed me that I was running Papercut mobility print, which must also start a DNS server.
